.github/workflows/gather-data.yml

```diff
@@ -0,0 +1,67 @@
+name: Update Data
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  fetch-plugin-data:
+    name: Fetch Plugin Data
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
```

Comment:
You probably also want to clone the submodule and update it. I think that is possible with the checkout action.

Reply:
Hmm, good point. I _think_ this can be done by:
```yml
jobs:
  fetch-plugin-data:
    name: Fetch Plugin Data
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v3
        with:
          submodules: true
```

I've also removed the manual `await $('git submodule update --remote');` call from both the plugin/theme index, as it probably isn't necessary anymore